README.md

```diff
@@ -148,12 +148,16 @@ ActiveRecord::Base.on_replica do
 end
 ```
 
-This will perform the query on the replica, and mark the returned instances as read only. There is also a shortcut for this:
+This will perform the query on the replica, and mark the returned instances as read-only. There is also a shortcut for this:
 
 ```ruby
 Account.on_replica.find_by_big_expensive_query
 ```
 
+If you do not want instances returned from replicas to be marked as read-only, this can be disabled globally:
+
+`ActiveRecordShards.disable_replica_readonly = true`
```

Comment:
Perhaps we should call it e.g. `disable_replica_readonly_records` instead. The replica _database_ should be read-only, and we don’t want users to be confused.

Reply:
Good idea, renamed!